jax/experimental/jax2tf/tests/primitives_test.py

```diff
@@ -109,6 +109,24 @@ def test_pad(self, harness: primitive_harness.Harness):
       raise unittest.SkipTest("pad with negative pad not supported")
     self.ConvertAndCompare(harness.dyn_fun, *harness.dyn_args_maker(self.rng()))
 
+  @primitive_harness.parameterized(primitive_harness.lax_top_k)
+  def test_top_k(self, harness: primitive_harness.Harness):
+    if (harness.params["k"] > harness.params["shape"][-1] or
```

Comment:
I do not think that we need to test this here. The goal is to test the conversion of primitive usage (lax.top_k_p.bind). The negative `k` will be rejected before we even get to use the primitive.

Reply:
I believe that we should keep these tests to make sure `lax.top_k` always fails in potential edge cases. Because the clamping behavior is not always consistent, it is important that we know if that invalid parameters are never silently passed to the function.